packages/taquito/src/constants.ts

```diff
@@ -51,7 +52,7 @@ export const protocols = {
   '013': [Protocols.PtJakart2],
   '014': [Protocols.PtKathman],
   '015': [Protocols.PtLimaPtL],
-  '016': [Protocols.PtMumbaii],
+  '016': [Protocols.PtMumbai2], // mumbai v2
```

Comment:
should we add the protocol to the other array like the ithaca one?

Reply:
I was thinking on doing that, but I think in this case it's okay to just choose one since PtMumbaii will never reach the public.